src/run_server.sh

```diff
@@ -187,5 +222,7 @@ set_variables
 update_folder_permissions
 apply_preinstall_config
 update_server
+first_start_config_ops
 apply_postinstall_config
```

Comment:
We need to remove this, this makes it impossible for users to change the configuration after initial config 
(Keeps overwriting settings on restart [needed to add and remove mods])

I have moved "apply_postinstall_config" into the first_start_config_ops function